package.json

```diff
@@ -42,6 +42,7 @@
     "@open-wc/testing-karma-bs": "^1.3.30",
```

Comment:
should be removed right?
- `@open-wc/testing-karma`
- `@open-wc/testing-karma-bs`

Reply:
also karma.config files

Reply:
also what do we do with test:bs and test:prune-snapshots and test:update-snapshots

Reply:
we don't use browserstack anyways... so remove

do we have failing snapshots tests? are we using shapshots? if no then remove